src/price/ProgressivePriceThreshold.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace hiqdev\php\billing\price;
+
+use hiqdev\php\units\Unit;
+use Money\Money;
+use hiqdev\php\units\Quantity;
+
+class ProgressivePriceThreshold
+{
+    public string $price;
```

Comment:
When I put here `"1"` – it's `0.01 $currency`, or `1 $currency`?

Reply:
1